docs/_sections/about.md

```diff
@@ -3,22 +3,45 @@ title: About This Toolkit
 description: Why this toolkit exists and how to use it.
 ---
 
-### NYPL Web Presence As A Public Space
+### Introduction
 
-Coming soon
+This toolkit aims to serve multiple purposes:
 
-### Modularity
+- facilitate the creation of new web user interfaces
+- ensure consistency in design and quality
+- serve as the main repository for documenting the web design efforts at NYPL Digital
```

Comment:
Add a 4th bullet point?
"- establish and promote universal design best practices"